includes/DefaultSettings.php

```diff
@@ -263,14 +263,14 @@
 		#
 		# In case `$smwgSparqlRepositoryConnector` is maintained with 'custom',
 		# the `$smwgSparqlCustomConnector` is expected to contain a custom class
-		# implementing the ncessary interface (see `SMWSparqlDatabase`).
+		# implementing the ncessary interface (see `GenericRepositoryConnector`).
```

Comment:
I suggest fixing the typo (ncessary => **necessary**) while we're touching this file, if not inconvenient

Reply:
Thanks, done.